.github/workflows/pr.yml

```diff
@@ -22,7 +22,6 @@ jobs:
           SKIP: ruff,ruff-format # In CI run ruff separately to only check, not fix.
       - name: ruff check
         run: |
-          uv lock --check
```

Comment:
I guess we had recently added this to fail CI, but I think adding to a pre-commit is more standard

Reply:
I added it, because it got out of sync. We need it too, fits best here.